modules/documentation/ex_suitability.md

```diff
@@ -1,39 +1,33 @@
 ---
-title: What is suitable documentation?
+title: Suitable documentation
 type: exercise
 order: 4
 ---
 
-# What is suitable documentation?
+# What is suitable documentation for "your" project?
 
-## Step 1: Choose a software project
+As a group, consider the project that you worked on for your homework assignment.
 
-Choose a software project for this exercise, preferably one you are familiar with.
 
-If you don't have a project in mind, you can use one of the following:
+## Type(s) of documentation
 
-- [ESMValTool](https://research-software-directory.org/software/esmvaltool)
-- [LitStudy](https://research-software-directory.org/software/litstudy)
-- [Haddock](https://research-software-directory.org/software/haddock3)
-- [worcs](https://cjvanlissa.github.io/worcs/index.html)
-- [democracy-topic-modelling](https://research-software-directory.org/software/democracy-topic-modelling)
+*Which type of of documentation currently exists? Is there anything missing/do you think anything is a bit over the top?*
 
-## Step 2: Discuss type(s) of documentation
-
-*Look at the software repository or page in the Research Software Directory*
-
-- Is documentation for this project important? Why?
-- How would you describe a useful documentation for this project?
-- How can you motivate your colleagues to contribute to the documentation?
-
-*What type of documentation seems appropriate to you?*
+It is not necessary to dive too deep into the codebase/repository for this. Answer based on what you can see 
 
+- README
+- Tutorial(s)
+- In-code documentation
+- API reference
+- Overview of components
 - User documentation
```

Comment:
@DaniBodor these three types of documentation that were mentioned here, are more or less three high level categories of documentation divided by the audience. The things you added seem to me to be a collection of 1) different formats/implementations of documentation [README, in-code docs, API ref/docstrings (could also add a web page to this list and a printed paper/book] and 2) a shape/narrative type [tutorial, overview of components, story/narrative]. I think it might be confusing to put them all in one list, but I do like the idea of looking at these different aspects of audience/implementation/shape. For example, you could have a tutorial that describes how to deploy the software written down in a README file, or you could have an overview of components and descriptions on how to use them written down in comments in the code (not the best user documentation, I agree). What do you think about this?

Reply:
I had put different "levels" of documentation into this list on purpose. Apart from the three high level ones, all the others are also part of the previous exercise as well as part of the "types of documentation" tab, so there should be plenty of info around.
I see how it could be confusing, but in my mind not so much so that it is preventative. I'm happy to change it though, if you disagree.